<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarAdmin</class>
 <widget class="QMainWindow" name="ModificarAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 241, 172)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>751</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
    </property>
    <property name="title">
     <string>DATOS PERFIL</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>CONTRASEÃ‘A: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCorreo_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>NOMBRES:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombres">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>APELLIDOS:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtApellidos">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>FECHA NACIMIENTO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCorreo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>CORREO:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtContraU">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFechaN">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>300</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCorreoU">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>MODIFICAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegresar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
